### PR TITLE
expand the readme to include basic usage and what's missing

### PR DESCRIPTION
**Pull request by cybojenix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+A collection of Terraform modules to provision the GovWiFi infrastructure on AWS.
+
+## Usage
+
+These modules are to be consumed by another Terraform config.
+Currently the main configs for GovWiFi are contained in the [`govwifi-build`][govwifi-build] repo.
+Please refer to this repo for execution instructions.
+
+In the future, each module should contain documentation around its inputs and outputs.
```
Comment (kjdchapman) [inline README.md]: Should this be in a TODO or an open issue? I think the README should describe the as-is state (think it's easier for it to fall out-of-date if it's trying to track progress).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+Currently, all security groups are managed in the private repo.
+
+These group relate to:
+
+- communication between services and their database
+- inter-service communication (RADIUS to backend services)
+- ensuring service ports (SSH) are only accessible via the bastion servers
+- ensuring the bastion servers are only accessible via known locations
+
+At a future data, these will be moved into this repo.
```
Comment (kjdchapman) [inline README.md]: Same comment as the other "TODO".

**Comment by kjdchapman** [comment]
Is there any pattern for what's in this repo and what's in build? 

As a naive reader (i.e., one that doesn't know much about terraform), it's not clear to me _why_ the two were split.

I do like this, though. Will happily tick once we've discussed.

**Comment by kjdchapman** [comment]
The conflict in this PR is down to me merging #197, sorry. It should be fairly easy to resolve, as #197 just added a single line. If not, talk to me!

File: README.md (hunk 1 above)
Comment (cybojenix) [reply README.md]: agreed, this should be in our trello board

File: README.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+## What's missing
+
+Currently there are some key pieces of the infrastructure missing from these modules, as they
+are managed in the private [`govwifi-build`][govwifi-build] repo.
+
+### Main Terraform config
+
+This ties all the modules together. In short, it:
+
+- Provides ann AWS provider to the modules, tied to a specific region
```
Comment (tommotaylor) [inline README.md]: spelling`ann`

File: README.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+- Configures any sensitive details, mainly around non-region specific values (e.g., RADIUS IP Addresses)
+- Controls which parts of the infrastructure go in which region
+
+### Security Groups
+
+Currently, all security groups are managed in the private repo.
+
+These group relate to:
```
Comment (tommotaylor) [inline README.md]: `these groups` with an s

**Pull request merged**